fruitsalad_test.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+import io
+import sys
+from fruitsalad import FruitSaladTool
+
+class TestFruitSalad(unittest.TestCase):
+    ''' Basic tests of fruit salad MongoDB log redacter '''
+
+    def setUp(self):
+        self.saved_stdout = sys.stdout
+        self.stdout = io.BytesIO()
+        sys.stdout = self.stdout
+
+    def tearDown(self):
+        sys.stdout = self.saved_stdout
+
+    def test_milliseconds(self):
+        ''' Timestamps should be printed correctly '''
+        tool = FruitSaladTool(arg_logfile='test/sample.ts.log', arg_seed='test')
```

Comment:
@kevinadi Might be safer to use a numeric constant for the `random.seed()`. The Python docs imply a non-numeric value might result in spurious failures:
> If a is not None or an int or a long, then hash(a) is used instead. Note that the hash values for some types are nondeterministic when PYTHONHASHSEED is enabled.